utils/upodman/Makefile

```diff
@@ -0,0 +1,106 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=upodman
+PKG_VERSION:=0.98.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/oskarirauta/upodman/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=c05eddfe584fcb4e7e1a8893d676356b6332fd3ee6d292793097d16dcc33e6b9
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/upodman
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Podman manager via Ubus
+  URL:=https://github.com/oskarirauta/upodman
+  DEPENDS:=+libjson-c +libstdcpp +libubox +libuci +libblobmsg-json +libubus +podman
+endef
+
+define Package/upodman/description
+  upodman extends ubus with bindings to podman and allows
+  controlling and statistics over pods and containers through
+  ubus.
+
+  upodman is written in C++ except for ubus parts which are
+  externed from C. It uses staticly linked libcurl for unix
+  socket operations, because system's default libcurl might
+  not support unix sockets.
+endef
+
+MAKE_VARS += \
+	BUILD_DIR="$(PKG_BUILD_DIR)"
+
+CURL_VERSION:=7.87.0
+CURL_PATH:=$(PKG_BUILD_DIR)/curl-$(CURL_VERSION)
+CONFIGURE_PATH:=curl-$(CURL_VERSION)
```

Comment:
No, I doubt you are going to bump upodman when new curl version is released

Reply:
If I see it fit, I will update upodman when curl is updated. Rarely updates have anything to do with unix-sockets as it's unlikely to change, and required http capabilities are quite basic.

Reply:
changing CURL_VERSION here does only effect on failed build, as source provided curl source is not found if version is changed. So it must go hand to hand with source of software. Version/extraction would not be relevant at all if curl would ship with software as already extracted, but that would just grow released source file enormously. If it would be under renamed file as curl.tar.gz for example and it would had been re-packed to unextract to curl instead of curl-7.87.0 directory, this would make things a lot different, but it would be more difficult to update the program and I see no point in re-packing curl, just to make package Makefile please someone's eye. Software was originally meant to be built as native only due to it targeting personal use, but on request I'm making it publicly available without requirement to build it natively.